contrib/openapi.yml

```diff
@@ -0,0 +1,170 @@
+swagger: "2.0"
+info:
+  title: Kostyor API
+  description: Aims & Launches Grenades
+  version: 1.0.0
+schemes:
+  - http
+consumes:
+  - application/json
+produces:
+  - application/json
+paths:
+  /environments:
```

Comment:
I would prefer this remain as `clusters` - since this has been used throughout our documentation.